osu.Game/Screens/Play/GameplayClockContainer.cs

```diff
@@ -63,17 +60,17 @@ public class GameplayClockContainer : Container
 
         private readonly FramedOffsetClock platformOffsetClock;
 
-        public GameplayClockContainer(WorkingBeatmap beatmap, IReadOnlyList<Mod> mods, double gameplayStartTime)
+        public GameplayClockContainer(ITrack track, WorkingBeatmap beatmap, IReadOnlyList<Mod> mods, double gameplayStartTime)
```

Comment:
Seeing that all call sites seem to be calling this with a non-null `track` I'd stick an annotation on it and eliminate all the null-checking cases.